src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -301,7 +301,7 @@ describe('StoryPromo Container', () => {
         <WrappedStoryPromo service="news" item={newItem} />,
       ).container;
       expect(newsContainer.getElementsByTagName('time')[0].innerHTML).toEqual(
-        'a minute ago',
+        '1 minute ago',
```

Comment:
As this new locale _also_ introduces relative time for hours, would it make sense to:
- Write a test, similar to this one, that tests relative time for hours?
- Rename this test to something more appropriate for minutes?

Reply:
Not sure - this is a front pages test, so possibly we should defer to that pod?

Reply:
I think it would be good to write a new one for hours and rename this for minutes, a new issue can be raised to do this.

Reply:
Might this be covered already by https://github.com/bbc/simorgh/pull/3894 ?